service/javadsl/client/src/main/java/com/lightbend/lagom/javadsl/client/CircuitBreakersConverter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.client;
+
+import scala.compat.java8.FutureConverters;
+import scala.concurrent.Future;
+import scala.runtime.AbstractFunction0;
+
+import java.util.concurrent.CompletionStage;
+import java.util.function.Supplier;
+
+public class CircuitBreakersConverter {
+
+  /**
+   * Temporary converter utility to convert {@link com.lightbend.lagom.internal.client.CircuitBreakers} to {@link CircuitBreakersPanel}
+   */
+  public static CircuitBreakersPanel toJavadslCircuitBreakerInvoker(com.lightbend.lagom.internal.client.CircuitBreakers circuitBreakers) {
+    return new CircuitBreakersPanel() {
```

Comment:
I adapted it accordingly. Added overloaded constructors (both Java and Scala) but public instead of private. `CircuitBreakersPanelImpl` are internal as well as `CircuitBreakersPanelInternal`, so not a big deal here.

Actually, I wanted to make this extra constructor public to align with the Scala API side.
We need the default constructor to accept `CircuitBreakersPanelInternal`, because overloaded constructors must call the default constructor in Scala.

```scala
class CircuitBreakersPanelImpl(circuitBreakersInternal: CircuitBreakersPanelInternal)
  extends CircuitBreakersPanel {

  def this(system: ActorSystem, config: CircuitBreakerConfig, metricsProvider: CircuitBreakerMetricsProvider) =
    // build a CircuitBreakersPanelInternal and call default constructor
    this(new CircuitBreakersPanelInternal(system, config, metricsProvider))
}
```
The other way around is not possible:


```scala
class CircuitBreakersPanelImpl(system: ActorSystem, config: CircuitBreakerConfig, metricsProvider: CircuitBreakerMetricsProvider) 
  extends CircuitBreakersPanel {

  def this(circuitBreakersInternal: CircuitBreakersPanelInternal) =
    ??? // can't call default constructor now unless all fields in CircuitBreakersPanelInternal are made public
}
```